lib/data/external/AwsClient.js

```diff
@@ -125,7 +125,7 @@ class AwsClient {
             Bucket: this._awsBucketName,
             Key: key,
             VersionId: dataStoreVersionId,
-            Range: range,
+            Range: range ? `bytes=${range[0]}-${range[1]}` : null,
```

Comment:
Not sure if it matters to AWS, but maybe use `undefined` instead of `null`.

Reply:
The caller was sending `null` before, so I used the same.